docs/content/docs/guide/concurrency.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Concurrency Flow
+---
+# Concurrency Flow
+
+![PAC Concurrency Flow](/images/concurrency-design.png)
```

Comment:
any chance embedding the source instead of the output ? https://hugo-book-demo.netlify.app/docs/shortcodes/mermaid/

Reply:
It's GTG for me